### PR TITLE
Issue #1028 Footer fix

### PR DESCRIPTION
**Pull request by dawoodkhan82**
# Description

Please include: 
* relevant motivation
* a summary of the change 
* which issue is fixed. 
* any additional dependencies that are required for this change.

Fixes: # (issue)
#1028
# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: ui/packages/app/src/Blocks.svelte [hunk 1]
```diff
@@ -204,6 +204,7 @@
 			return dep.filter((_id) => _id !== id);
 		});
 	}
+	console.log("Blocks.svelte Hit")
```
Comment (pngwn) [inline ui/packages/app/src/Blocks.svelte:207]: Leftover comment.

File: ui/packages/app/src/main.ts [hunk 2]
```diff
@@ -102,6 +102,7 @@ window.launchGradio = (config: Config, element_query: string) => {
 		}
 		let session_hash = Math.random().toString(36).substring(2);
 		config.fn = fn.bind(null, session_hash, config.root + "api/");
+		console.log("new Blocks Hit!")
```
Comment (pngwn) [inline ui/packages/app/src/main.ts:105]: Leftover comment.

File: ui/packages/app/src/main.ts [hunk 3]
```diff
@@ -130,6 +131,7 @@ async function get_config() {
 
 if (window.gradio_mode == "app") {
 	get_config().then((config) => {
+		console.log("Launching Gradio")
```
Comment (pngwn) [inline ui/packages/app/src/main.ts:134]: Leftover comment.

**Review by pngwn — APPROVED** [review]
Few left over comments, otherwise looks good. 👍

File: ui/packages/app/src/Blocks.svelte (hunk 1 above)
Comment (dawoodkhan82) [reply ui/packages/app/src/Blocks.svelte:207]: Oh sorry was supposed to close this PR. as it doesn't actually solve the issue.

**Pull request closed without merging**